Misc/NEWS.d/next/Library/2024-06-19-19-53-42.gh-issue-41431.gnkUc5.rst

```diff
@@ -0,0 +1,2 @@
+Add :meth:`datetime.time.strptime` and :meth:`datetime.date.strptime`.
+ (Contributed by Wannes Boeykens in :gh:`120752`.)
```

Comment:
Aaaaaah I deleted my suggestion instead of editing it. Just wanted to say that you have one space before `(Contributed`.

And the next comment was:

> In general, I only put the gh issue in the What's New entry and not in the CHANGELOG but I'm not sure if this is how it should be done (usually, the release manager would normalize the changelogs but the previous changelogs did not have those kind of additions. In general, there is just the "Patch/Contributed by XXX" without parentheses).

Reply:
I'll just do what vstinner does: https://github.com/python/cpython/pull/120059/files.